.circleci/config.yml

```diff
@@ -24,9 +24,9 @@ references:
     VAULT_BINARY_VERSION: 1.9.4
     GO_VERSION: 1.18.1
   envoy-versions: &supported_envoy_versions
-    - &default_envoy_version "1.19.5"
-    - "1.20.4"
-    - "1.21.3"
+    - &default_envoy_version "1.21.4"
```

Comment:
did we skip 1.20 here?

Reply:
No, this is a little confusing because 1.22 was accidentally removed then the refactor to add this config was added in https://github.com/hashicorp/consul/pull/13610. If anything, since we should be supporting 4 versions, I should probably add 1.20 to the list.